spacy/tests/doc/test_json_doc_conversion.py

```diff
@@ -0,0 +1,155 @@
+import pytest
+from spacy.tokens import Doc, Span
+
+
+@pytest.fixture()
+def doc(en_vocab):
+    words = ["c", "d", "e"]
+    pos = ["VERB", "NOUN", "NOUN"]
+    tags = ["VBP", "NN", "NN"]
+    heads = [0, 0, 0]
```

Comment:
Since 0 is the default, let's use at least one non-zero head for the test example?

Reply:
Included in next commit.